samples/snippets/noxfile.py

```diff
@@ -39,31 +39,29 @@
 
 TEST_CONFIG = {
     # You can opt out from the test for specific Python versions.
-    'ignored_versions': ["2.7"],
-
+    "ignored_versions": ["2.7"],
```

Comment:
Should we simply be eliding 2.7 upstream of this noxfile?

Reply:
I'm not very familiar with structuring testing in Python, but I think the way the tests are set up means it needs to be specified in the separate noxfiles.

Reply:
TIL the word elide. This noxfile is for everything in samples/snippets and will skip 2.7. If you need to skip it for other samples, you'll need to add a noxfile_config specifying what you want to omit in that directory [here's an example from Dataflow](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/c68fd4a2373a3258f7f4e2b8c4eecbb771eaf465/dataflow/gpu-workers/noxfile_config.py#L25)

Reply:
The noxfile is generated by owlbot, but the config is your manual override for it

Reply:
Okay, thank you for clarifying @leahecole. It looks like the noxfiles for all the samples have 2.7 in `ignored_versions`, so am I correct that we're good as is? (ie, samples/quickstart/noxfile.py)

Reply:
If you want to ignore 2.7, you are good as is! 😄